v2/go.mod

```diff
@@ -1,33 +1,29 @@
 module github.com/cloudevents/sdk-go/v2
 
 require (
-	cloud.google.com/go v0.40.0
-	contrib.go.opencensus.io/exporter/prometheus v0.1.0
-	github.com/Azure/go-amqp v0.12.7
-	github.com/Shopify/sarama v1.19.0
-	github.com/cloudevents/sdk-go v1.1.2
-	github.com/cucumber/godog v0.9.0
-	github.com/cucumber/messages-go/v10 v10.0.3
+	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/golang/protobuf v1.3.2 // indirect
 	github.com/google/go-cmp v0.4.0
 	github.com/google/uuid v1.1.1
-	github.com/gorilla/mux v1.7.3
 	github.com/hashicorp/golang-lru v0.5.3 // indirect
 	github.com/kelseyhightower/envconfig v1.4.0
```

Comment:
Is that required? I thought this is only for samples

Reply:
it should be... let me look

Reply:
ah, it comes from cmd/tools... let me add a mod for that.